kptncook/__init__.py

```diff
@@ -56,7 +56,7 @@ def save_todays_recipes():
 
 
 def get_mealie_client() -> MealieApiClient:
-    client = MealieApiClient(settings.mealie_url)
+    client = MealieApiClient(settings.mealie_url, settings.kptncook_api_key)
```

Comment:
Hmm, this is needed to fetch images from kptncook, right?
Maybe just pass the images as part of the recipe, to avoid coupling the client
to kptncook?

Reply:
Yes, exactly. I also thought about that, cause I also didn't want to merge these two parts, but I didn't want to extra touch the stored kptncook's json... at least it seems like the simplest solution at 2 in the morning, but ofc it should also be possible to download the images and modify the recipe json before passing it to the mealie adapter